css/style.css

```diff
@@ -67,45 +67,55 @@ html {
   gap: 5rem;
 }
 
-.main-nav-link,
-.main-nav-link:link,
-.main-nav-link:visited,
-.main-nav-link:active {
```

Comment:
Why did you remove the :link, :visited, and :active pseudo-classes?

Reply:
I decided that adding them at this stage - I am ahead of the curve. Is it worth returning these properties?

Reply:
It's standard to have them for links even if all the styles are the same.

Reply:
Let's finish the most important part with layout and positioning and then tune links as a next step